src/components/Tooltip/index.js

```diff
@@ -0,0 +1,48 @@
+import { Typography, Button, Divider } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(({ typography }) => ({
+  tagline: {
+    margin: `${typography.pxToRem(40)} 0 ${typography.pxToRem(20)}`,
+  },
+  divider: {
+    height: typography.pxToRem(2),
+  },
+}));
+
+function Banner({ tagline, ctaText, ...props }) {
```

Comment:
```suggestion
function TooltipBanner({ tagline, ctaText, ...props }) {
```

Reply:
mmh 🤔  ! I am already in the Tooltip folder, why should I name one of its component `Tooltip`Banner  ..tooltip again.

Reply:
@kilemensi

Reply:
I don't know @KhadijaMahanga , you're confusing the hell out of me; what is the name of the component? Tooltip or TooltipBanner?

Reply:
ahh, I see why you're confused, old age?.. But yes, it is confusing..so changed the folder name to tooltipbanner

Reply:
Where is HR when you need them? This is straight up workplace discrimination right here!